src/lib/components/console/Console.svelte

```diff
@@ -0,0 +1,174 @@
+<script lang="ts">
+	import ConsoleLine from "./ConsoleLine.svelte";
+
+	enum Tabs {
+		Frontend,
+		Backend,
+		All,
+	}
```

Comment:
Maybe this enum should be in its own file so that it can be used by other components?